python/magika/magika.py

```diff
@@ -40,7 +40,7 @@
     StatusOr,
 )
 
-DEFAULT_MODEL_NAME = "draft_standard_v2"
+DEFAULT_MODEL_NAME = "standard_v2_0"
```

Comment:
To replicate this change in Rust, you need to update the `rust/gen/model` symbolic link to point to the new directory. You'll need to run the sync script, similarly to the other PR because the model name changed.